data-preprocessing/JSON  Section/README.md

```diff
@@ -0,0 +1,12 @@
+# How to use
+
+## Useful Information
+The json files are of two forms. The structure of the Algebra.json is slightly different from Chemistry 2e.json and College Physics 2e.json. As such slightly different block of code handle the return of the sections to account for these differences.
+
+## Function
+The main.py file accepts a path to json files of parsed books. These files contain portions called sections. The file then returns all sessions alongside their page numbers in a csv file format; csv file for each json file.
+
+## Intructions
+- Open the main.py file
+- Replace the PATH variable with the absolute path location of the json files
```

Comment:
typically you would either get user input via the command line arguments that are parsed in or you would have a separate input file that the user can supply to the script. Ideally you don't want folks modifying their scripts in order to use them.

Reply:
@PyJim You may want to consider using a json instruction file here as I did in my previous script here. See https://github.com/nsmq-ai/nsmqai/tree/main/data-preprocessing/video-download-upload